.gitignore

```diff
@@ -13,11 +13,13 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
Strange that you added this because line 21 (was line 20) already had it.

Reply:
oops, i'll take that out!